operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/reconciler/Reconciler.java

```diff
@@ -28,7 +28,7 @@
    *         finalizer to indicate that the resource should not be deleted after all, in which case
    *         the controller should restore the resource's state appropriately.
    */
-  default DeleteControl deleteResource(R resource, Context context) {
+  default DeleteControl deleteResources(R resource, Context context) {
```

Comment:
Asked this on the releated issue, we can discuss this on community meeting today. The plural is because is deletes the dependent resources, not the resource itslef - at least that is what it should do. Agree that this is not very explicit. I would just name it `cleanup` and document it.

Reply:
renamed.